docs/pages/pmd/userdocs/extending/metrics_howto.md

```diff
@@ -244,8 +244,8 @@ the Java framework but it's symmetrical in the Apex framework.
   patterns you may want to look at:
   * *Visitor metrics:* Those metrics use one or more AST visitor to compute their
     value. That's especially good to implement metrics that count some kind of node,
-    e.g. [NPath complexity](https://github.com/pmd/pmd/blob/master/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/metrics/internal/NpathMetric.java)
-    or [NCSS](https://github.com/pmd/pmd/blob/master/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/metrics/internal/NcssMetric.java).
+    e.g. [NPath complexity](https://github.com/pmd/pmd/blob/master/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/metrics/impl/NpathMetric.java)
```

Comment:
you seem to just have reverted this, but this location is invalid now. It should say `https://github.com/pmd/pmd/blob/master/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/metrics/impl/internal/NpathMetric.java`

Reply:
NPathMetric is in the `impl` package though, not `impl/internal`. If we internalize it I expect it to be in  `metrics/internal` without an `impl` segment